static/app/components/search/index.tsx

```diff
@@ -38,17 +37,17 @@ type InputProps = Pick<
  *  itemProps: props that should be spread for root item
  * }
  */
-type ItemProps = {
-  item: Item;
+interface ItemProps {
+  item: Result['item'];
   matches: Result['matches'];
   index: number;
   highlighted: boolean;
   itemProps: React.ComponentProps<typeof SearchResultWrapper>;
-};
+}
 
 // Not using typeof defaultProps because of the wrapping HOC which
 // causes defaultProp magic to fall off.
-const defaultProps = {
+export const DefaultSearchProps = {
```

Comment:
we usually use capital letters only in the case of a `type/interface` and `React Component`... in this case should we keep the first letter lowercase?

Reply:
I think this is a weird case - you would never usually export DefaultProps value like this... I'm just going to get rid of them